packages/code-studio/src/settings/ColumnSpecificSectionContent.tsx

```diff
@@ -35,31 +33,11 @@ import {
   RootState,
   WorkspaceSettings,
 } from '@deephaven/redux';
-import { assertNotNull, DbNameValidator, TimeUtils } from '@deephaven/utils';
+import { assertNotNull } from '@deephaven/utils';
 import './FormattingSectionContent.scss';
 import type { DebouncedFunc } from 'lodash';
-
-const log = Log.module('ColumnSpecificSectionContent');
-
-type FormatterItem = {
-  columnType: string;
-  columnName: string;
-  format: Partial<TableColumnFormat>;
-  id?: number;
-  isNewRule?: boolean;
-};
-
-type FormatOption = {
-  defaultFormatString?: string;
-};
-
-function isFormatStringFormat(
-  format: Partial<TableColumnFormat>
-): format is Pick<TableColumnFormat, 'formatString'> {
-  return (
-    (format as Pick<TableColumnFormat, 'formatString'>).formatString != null
-  );
-}
+import SettingsUtils from './SettingsUtils';
```

Comment:
You could also import just the functions you use directly, so you don't need to call `SettingsUtils.*` each time, e.g. 

```
import { focusFirstInputInContainer } from './SettingsUtils';
...
  componentDidMount(): void {
    focusFirstInputInContainer(this.containerRef.current);
  }
```